sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -35,9 +35,9 @@ To update generated files for chat service, run the following command
 ### Code generation settings
 
 ```yaml
-tag: package-chat-2023-11-07
+tag: package-chat-2024-03-07
```

Comment:
Add the version of autorest used to ensure everyone uses the same version to regenerate code for this library.
```suggestion
tag: package-chat-2024-03-07
use: '@autorest/java@4.1.25'
```

Reply:
Regenerated with @autorest/java@4.1.25